workers/scanner/src/test/kotlin/OrtServerScanResultStorageTest.kt

```diff
@@ -101,7 +73,7 @@ class OrtServerScanResultStorageTest : WordSpec() {
 
                 verifyAssociatedScanResults(scannerRun, scanResult)
 
-                scanResultStorage.read(provenance, scannerMatcher) shouldBe listOf(scanResult)
+                scanResultStorage.read(provenance.mapToOrt(), scannerMatcher) shouldBe listOf(scanResult)
```

Comment:
Why these changes with `mapToOrt` ? They seem unrelated to this commit.

Reply:
`mapToOrt` is now needed, since the data types have changed. The tests have to deal with both ORT types and ORT Server types. Test data is now created using the model of ORT Server, but the scan result storage expects types from ORT's model.